submissions/mustbefail/friends-app/src/js/services.js

```diff
@@ -0,0 +1,54 @@
+export const API_REQ = `https://randomuser.me/api/?results=30&seed=a123bc&nat=us,dk,fr,gb&inc=gender,name,registered,dob,location,picture,phone,email`;
+
+const fullName = (user) => `${user.name.first} ${user.name.last}`.toLowerCase();
+const convertDate = (date) => new Date(date);
+
+const sorters = {
+  abc: (a, b) => fullName(a).localeCompare(fullName(b)),
+  zyx: (a, b) => fullName(b).localeCompare(fullName(a)),
```

Comment:
Meaningless names `abc` and `zyx`. Please, give them more meaningful names.
Also, it is the same function ;) Only order of arguments matters